sale_invoice_plan/models/sale_invoice_plan.py

```diff
@@ -112,6 +112,7 @@ def _compute_new_invoice_quantity(self, invoice_move):
             else:
                 plan_qty = order_line.product_uom_qty * (percent / 100)
                 prec = order_line.product_uom.rounding
+                plan_qty = round(plan_qty, precision_rounding=prec)
```

Comment:
```suggestion
                plan_qty = float_round(plan_qty, precision_rounding=prec)
```

and don't forget to import it.

this is why CI is red

Reply:
done